### ISSUE TITLE
Windows10 version

### ISSUE THREAD

**Issue by TheWebs** [jaquadro/NBTExplorer#44]
Minecraft win 10 version generates a mcworld file which when decompressed with winrar give us level.dat and other stuff. level.dat can't be opened by your tool, it fails to open.

Here is my world for example if u wish to try:
http://www.mediafire.com/download/k6uy4fjvje89mcd/Our+World.mcworld

**Comment by DaemonUmbra** [comment]
Or alternately you can play the multi-platform Java version of minecraft which IS supported by this tool.

**Comment by grouNDfx** [comment]
I agree with @TheWebs I would love to see this application support the "pocket edition" saves

**Comment by Dolphin2Point1** [comment]
Bedrock Edition is not supported, as the format is different. Maybe you could look for a different tool that can modify Bedrock Edition
